dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/fat/AcmeValidityAndRenewTest.java

```diff
@@ -667,4 +908,51 @@ public void autoRenewWithErrorAndRecovery() throws Exception {
 		}
 	}
 
+	/**
```

Comment:
Isn't this method already somewhere else? I figured we did a renew in AcmeCaRestHandlerTest? Can we use that method, or are they different functionality-wise?